examples/02_decoding/plot_haxby_anova_svm.py

```diff
@@ -61,10 +61,10 @@
 # Define the dimension reduction to be used.
 # Here we use a classical univariate feature selection based on F-test,
 # namely Anova. We set the number of features to be selected to 500
```

Comment:
As stated in the issue, this sentence needs to be edited "We set the number of features to be selected to 500" and replaced with "When doing full-brain analysis, it is better to use SelectPercentile, keeping 5% of voxels (because it is independent of the resolution of the data)."